code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -758,6 +757,8 @@
 		/obj/item/reagent_containers/hypospray/advanced/oxycodone,
 		/obj/item/storage/firstaid/adv,
 		/obj/item/clothing/glasses/hud/health,
+		/obj/item/storage/pouch/medkit/corpsman,
+		/obj/item/clothing/tie/storage/white_vest/medic,
```

Comment:
don't add to essentiel set, the medkit is already in the clothes corps specific.
Same for the white vest, it's already there